kymatio/visuals.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+"""Convenience visual methods."""
```

Comment:
i'd rather leave this file outside of the PR for now. but let's keep it for later

Reply:
The plan is to drop it right before merging into `dev`, but keep it so we can visualize locally